Tests/UnitTests/Purchasing/Purchases/PurchasesDiagnosticsTrackingTests.swift

```diff
@@ -24,14 +24,18 @@ class PurchasesDiagnosticsTrackingTests: BasePurchasesTests {
         self.setupPurchases()
     }
 
+    #if os(iOS)
```

Comment:
Nitpick but should this be `#if os(iOS) || VISION_OS`? Seems to be the visibility for the original method I believe.

Reply:
You're right. Thanks for pointing out!

Reply:
Done in https://github.com/RevenueCat/purchases-ios/pull/4897/commits/a00530f04ac2dfbc7b92e8745328af44dde716e0

BTW, used `os(visionOS)` instead of `VISION_OS`, as we already use it like this in some places and it should be the way to go from now on, as it's the official way. Eventually I think we should remove `VISION_OS`